index.html

```diff
@@ -4891,6 +4891,20 @@ <h3>Status</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h3>Schema</h3>
+
+        <p>
+If the <code>credentialSchema</code> property is available, the schema of a
+<a>verifiable credential</a> is expected to be evaluated by the <a>verifier</a>
+according to the <code>credentialSchema</code> <a>type</a> definition for the
+<a>verifiable credential</a> and the <a>verifier's</a> own schema evaluation
+criteria. For example, if the <a>credentialSchema</a>'s' <a>type</a> value is
```

Comment:
```suggestion
criteria. For example, if the <code>credentialSchema</code>'s' <a>type</a> value is
```

Reply:
You're switching between `<code></code>` markup, and links to sections normatively defining the term, and it's not clear why you're doing that. Try to be consistent. If you feel like you need to refer to the sections on type, use the `<a></a>` setup, if you just want to refer to the property used in the document, use the `<code></code>` markup (with a fair warning that we're moving from `<code></code>` to ```` markup (the latter being ReSpec syntactic sugar for the former).